### PR TITLE
Show 'Client.solvers' in docs

### PR DESCRIPTION
**Pull request by randomir**

### CODE REVIEW

File: docs/reference/resources.rst [hunk 1]
```diff
@@ -43,6 +43,7 @@ Methods
    :toctree: generated
 
    client.Client.from_config
+   client.Client.solvers
```
Comment (JoelPasvolsky) [inline docs/reference/resources.rst]: I already see this https://docs.ocean.dwavesys.com/projects/cloud-client/en/latest/reference/resources.html
image

File: dwave/cloud/client.py [hunk 2]
```diff
@@ -737,9 +737,9 @@ def get_solver(self, name=None, features=None, refresh=False):
 
             features (dict, optional):
                 Dictionary of features this solver has to have. For a list of
-                feature names and values, see: :meth:`Client.solvers`. Specifying solver
-                name overrides features. To include (possibly partial) name match requirement,
-                use the ``name`` feature inside ``features`` dictionary.
+                feature names and values, see: :meth:`~dwave.cloud.client.Client.solvers`.
+                Specifying solver name overrides features. To include (possibly partial)
```
Comment (JoelPasvolsky) [inline dwave/cloud/client.py]: Specifying the 'name' parameter overrides the 'feature' parameter.

File: dwave/cloud/client.py [hunk 3]
```diff
@@ -737,9 +737,9 @@ def get_solver(self, name=None, features=None, refresh=False):
...
+                name match requirement, use the ``name`` feature inside ``features`` dictionary.
```
Comment (JoelPasvolsky) [inline dwave/cloud/client.py]: To require a (full or partial) name match, use the 'features` parameter to specify a value for its 'name' key.

File: docs/reference/resources.rst (hunk 1 above)
Comment (randomir) [reply docs/reference/resources.rst]: Yes, because I already merged this PR. :)

File: dwave/cloud/client.py (hunk 2 above)
Comment (randomir) [reply dwave/cloud/client.py]: Thanks, will change in the next PR.

File: dwave/cloud/client.py (hunk 3 above)
Comment (randomir) [reply dwave/cloud/client.py]: Changed! Tnx.

**Pull request merged**